tools/run_local_ironic.sh

```diff
@@ -181,6 +185,16 @@ if [ -n "$MARIADB_KEY_FILE" ]; then
      CERTS_MOUNTS="${CERTS_MOUNTS} -v ${MARIADB_KEY_FILE}:/certs/mariadb/tls.key "
 fi
 
+if [ -n "$IPXE_CACERT_FILE" ]; then
+     CERTS_MOUNTS="${CERTS_MOUNTS} -v ${IPXE_CACERT_FILE}:/certs/ca/ipxe/tls.crt "
```

Comment:
One day (not in this PR) these should be written as arrays and quotes, these chained strings will break on whitespace.

Reply:
I agree, at some point every similar place should be refactored to use arrays.